### PR TITLE
Add WALKMAN

### LINKED ISSUE #4117: Add walkman logo
It's old logo but I think it would be great to add walkman logo

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
WALKMAN

**Issue:** Closes #4117
**Alexa rank:** n/a (see below)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
As a child of the 80s who loved his Walkmans, I _had_ to take this one for myself! 😆

It _is_ still an active brand but the question is: is it active and popular enough to warrant inclusion in our library.

### CODE REVIEW

**Comment by adamrusted** [comment]
Looks good to me - I'm happy for it to be added to the library, but will await another @simple-icons/maintainers approval before merging, just to be sure.

**Review by service-paradis — APPROVED** [review]
So many memories! 🚶 🎧 🎶 
Thanks @PeterShaggyNoble!

**Pull request merged**
